### DISCUSSION TITLE
Doodba and Odoo RPC, Connection refused on Mass Invoice import

### DISCUSSION THREAD

**Discussion by bosd** [Tecnativa/doodba#496] [Q&A] [answered]
How does one use doodba in combination with odoorpc?
When trying to connect, keep getting connection refused.

I'm using the invoice_import script:
https://github.com/OCA/edi/tree/14.0/account_invoice_import/scripts

Running the script on a local machine with odoorpc installed.

[ERROR] Failed to connect to Odoo. Error: <urlopen error [Errno -2] Name or service not known.

Settings:
ODOO_SERVER="https://myodooinstance.com"
ODOO_PORT="443"
ODOO_DB="PROD"
ODOO_LOGIN="admin"
ODOO_PASSWORD="mysecret"

**Comment by pedrobaeza** [comment, marked as answer]
Database name should be `prod`, not `PROD`, and Odoo server host should be without `https://`. Check https://pythonhosted.org/OdooRPC/faq.html about connecting to SSL instance.

**Comment by bosd** [comment]
Many thanks!!
It's working now. Issues I've encountered were not related to doodba.

Some lessons learned:
- The mass invoice script used `""` for the password field. Somehow it couldnt handle the special characters in my password. Placing the password between `''` solved it.
- On python 3.7 it is working, but odoo rpc has compatability issues with python 3.10.
Will open a issue/pr there.
